spec/unit/messages/service_instances_list_message_spec.rb

```diff
@@ -56,5 +60,23 @@ module VCAP::CloudController
         expect(message.errors[:base]).to include("Unknown query parameter(s): 'foobar'")
       end
     end
+
+    describe 'validations' do
+      context 'names' do
+        it 'validates names is an array' do
+          message = ServiceInstancesListMessage.new names: 'tricked you, not an array'
+          expect(message).to be_invalid
+          expect(message.errors[:names]).to include('must be an array')
+        end
+      end
+
+      context 'space guids' do
+        it 'validates app_guids is an array' do
```

Comment:
space_guids

Reply:
Oops, fixed.

Reply:
Changes for this included in #1033 and can be viewed here: https://github.com/cloudfoundry-incubator/cloud_controller_ng_sapi/commit/ff8276e5d201e573df03640a0f6865288b2d55af#diff-58747ec5e09fb7b688e999c46413b076R74